### DISCUSSION TITLE
Fragment-GC bias files

### DISCUSSION THREAD

**Discussion by Ray6283** [COMBINE-lab/salmon#884] [Q&A (bulk specific)]
In https://salmon.readthedocs.io/en/latest/file_formats.html#quantification-file, it mentions that If "Salmon was run with fragment-GC bias correction enabled, the auxiliary directory will contain two files named expected_gc.gz and observed_gc.gz"

How can we run Salmon with fragment GC bias correction?
Thanks.

**Comment by rob-p** [comment]
Pass the `--gcBias` flag to the `quant` command, as described [here](https://salmon.readthedocs.io/en/latest/salmon.html#gcbias).
